xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -525,8 +538,8 @@
         <Parameter Name="item" Type="System.Text.RegularExpressions.Capture" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="item">The object to add to the collection.</param>
+        <summary>Adds an item to the collection.</summary>
```

Comment:
All the regex collections are read-only. This method will always throw a NotSupportedException.

1. Do we want to mention that in the summary?
2. Should we document the `<exception` in all the places it is thrown?

Reply:
1. Good catch. The type summary (line 58 in this file) does not mention it is read-only, but the type remarks (line 63) does mention it. The problem with this is that the user will not know it is read-only when reading the type summary in Intellisense. I say we also mention it is read-only in the type summary (line 58).

2. Yes, all obvious exceptions should be documented.

Reply:
1. Done.

2. Done as well. I text searched for all the exceptions and documented everything I found.

Reply:
This still needs to be address, @pgovind. I think my explanation may have been confusing - the summary for this API should also mention that it is not possible to add items to the collection because it is read-only, so the method throws, always.